README.md

```diff
@@ -44,6 +44,45 @@ Alternatively using `cargo install`:
     (while sleep 5; do echo; done) | target/release/metamath-knife --timing --jobs 4 --split --repeat --trace-recalc --verify set.mm/set.mm
     # then make small changes to the beginning, end, or middle of the DB and observe how behavior changes
 
+Here is the command line help, which gives an idea of the options available:
+```
+A Metamath database verifier and processing tool
+
+USAGE:
+    metamath-knife [FLAGS] [OPTIONS] <DATABASE>
+
+FLAGS:
+        --debug                Activates debug logs, including for the grammar building and statement parsing
+        --free                 Explicitly deallocates working memory before exit
+    -g, --grammar              Checks grammar
+    -h, --help                 Prints help information
+    -O, --outline              Shows database outline
+    -p, --parse-stmt           Parses all statements according to the database's grammar
+    -t, --parse-typesetting    Parses typesetting information
+    -F, --print-formula        Dumps the formulas of this database
+    -G, --print-grammar        Prints the database's grammar
+        --dump-typesetting     Shows typesetting information
+        --repeat               Demonstrates incremental verifier
+        --split                Processes files > 1 MiB in multiple segments
+        --timing               Prints milliseconds after each stage
+        --trace-recalc         Prints segments as they are recalculated
+    -V, --version              Prints version information
+    -v, --verify               Checks proof validity
```

Comment:
Isn't `-v` standard for `--version` ? (i.e., isn't it better to swap `-v` and `-V` above?)

Reply:
Interestingly, [`-V` is the default](https://docs.rs/clap/latest/clap/builder/struct.Command.html#method.version) for `clap`, the command line parsing library we use.

Reply:
I think we could also use a '-v/--verbose' option, though that's an annoyingly incompatible change.

Reply:
Can we resolve this remark?

Reply:
yes you can resolve